### PR TITLE
*: add backup creation time

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: pkg/spec/backup.go [hunk 1]
```diff
@@ -71,6 +73,9 @@ type BackupServiceStatus struct {
 }
 
 type BackupStatus struct {
+	// Creation time of the backup.
+	Created time.Time `json:"created"`
```
Comment (hongchaodeng) [inline pkg/spec/backup.go]: createdTime?

Reading "created" from json would be hard to interpret.

**Comment by hongchaodeng** [comment]
xref: https://github.com/coreos/etcd-operator/issues/696

File: pkg/spec/backup.go (hunk 1 above)
Comment (xiang90) [inline pkg/spec/backup.go]: OK.

**Comment by xiang90** [comment]
PTAL

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**